server/bleep/src/ctags.rs

```diff
@@ -51,7 +51,7 @@ fn find_files(path: &Path) -> Vec<String> {
         })
         .filter(|de| matches!(de.file_type(), Some(ft) if ft.is_file()))
         .map(|de| {
-            dunce::canonicalize(de.path())
+            crate::canonicalize(de.path())
```

Comment:
@llogiq i am not sure if there is much to be gained here by conditionally compiling dunce as a dependency. internally, it uses a cfg flag to point to `fs::canonicalize` on non-windows targets (IIRC), and other targets should end up with zero extra compilation. for what its worth, dunce is a 100 line library, we might as well vendor it in if required.

Reply:
I know it's a small crate, but adding it unconditionally means we download and compile it everywhere. Yes, even macros are compiled. So even if it's just a second to download it and another half to compile, we end up winning one and a half second per non-windows build. A small win, yes, but those tend to add up.

Reply:
makes sense.